modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -486,6 +488,31 @@ class NDB_Form_Imaging_Browser extends NDB_Form
     }
 
     /**
+     * Gets candidate's scans with the same acquisitionID.
+     *
+     * @return string of FileIDs, comma separated
+     */
+    function _getFilesAcrossTimepoints($initialFileID, $initialFileAcquisitionID){
+
+        $selectResults = $this->DB->pselect("SELECT FileID FROM files ".
+            "WHERE AcquisitionProtocolID =:initialFileAcquisitionID ".
+            "AND SessionID in (SELECT s2.ID FROM session s1 RIGHT JOIN session s2 ".
+            "ON s1.CandID = s2.CandID WHERE s1.ID =:initialFileID)",
```

Comment:
can't this just be:

`SELECT FileID from files f JOIN session s ON (s.ID=f.SessionID) WHERE AcquisitionProtocolID=:initialFileAcquisitionID AND s.CandID=:initialCandID`

to make the query a lot simpler? (you'd have to pass in the initialCandID instead of the initialFileID, obviously..)